src/resources/postcss/components/attendees-list/_attendees.pcss

```diff
@@ -35,4 +35,10 @@
 		align-self: center;
 		flex-grow: 1;
 	}
+
+	.tec-tickets__attendees-list-item-attendee-details-name {
+		font-size: var(--tec-font-size-3);
+		font-weight: var(--tec-font-weight-bold);
```

Comment:
We should use the [classes from the design system](https://github.com/the-events-calendar/tribe-common/blob/master/src/resources/postcss/base/full/typography/_body.pcss) to give the font-size and font-weight. The classes should be in the HTML element, and if there's something that's not covered by the class of the design system (maybe adjusting line height for example), we add it here. But we have classes with this size, and font-weight.